src/daemon/daemon_rpc.cpp

```diff
@@ -59,6 +62,17 @@ auto make_server(const std::string& server_address, multipass::Rpc::Service* ser
     if (server == nullptr)
         throw std::runtime_error("Failed to start multipass gRPC service at " + server_address);
 
+    auto group = getgrnam("adm");
+    if (!group)
+        throw std::runtime_error("Could not determine group id for 'adm'.");
+
+    auto socket_path = QString::fromStdString(server_address).section("unix:", 1, 1).toStdString();
+    if (chown(socket_path.c_str(), 0, group->gr_gid) == -1)
+        throw std::runtime_error("Could not set ownership of the multipass socket.");
+
+    if (chmod(socket_path.c_str(), S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP) == -1)
+        throw std::runtime_error("Could not set permissions for the multipass socket.");
+
```

Comment:
This section can go in daemon_main.cpp, but since it's leaking the fact that the server address is a socket...it belongs somewhere in platform.

Reply:
Yeah, I was thinking about that last night:)  I'll work on moving this section.